src/views/Releases/ListReleases/index.jsx

```diff
@@ -276,9 +474,76 @@ function ListReleases(props) {
     // space below the card (margin)
     height += theme.spacing(6);
 
+    if (release.scheduledChange && release.scheduledChange.sc_id) {
+      // divider
+      height += theme.spacing(2) + 1;
+
+      // Scheduled changes row, which includes some text, a button, and a chip.
+      // Which one is largest depends on platform/browser.
+      height += Math.max(subtitle1TextHeight(), theme.spacing(3), buttonHeight);
+
+      if (Object.keys(release.scheduledChange.required_signoffs).length > 0) {
+        const requiredRoles = Object.keys(
+          release.scheduledChange.required_signoffs
+        ).length;
+        const nSignoffs = Object.keys(release.scheduledChange.signoffs).length;
+        // Required Roles and Signoffs are beside one another, so we only
+        // need to account for the one with the most items.
+        const signoffRows = Math.max(requiredRoles, nSignoffs);
+
+        // Padding above the summary
+        height += theme.spacing(2);
+
+        // The "Requires Signoff From" title and the margin beneath it
+        height += listSubheaderTextHeight + theme.spacing(0.5);
```

Comment:
I think we are using an unusual subheader in the sense we are hard-coding the line-height. Instead, we could something along the lines of:

```jsx
height += signoffSummarylistSubheaderTextHeight + theme.spacing(0.5);

// elementsHeight.js
signoffSummarylistSubheaderTextHeight: remToPx(theme.typography.subtitle1.fontSize) * 1.5
```